src/index.html

```diff
@@ -4,13 +4,52 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      "
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form form--place"
+      action="/"
+      method="post"
+      data-qa="big"
+    >
+      <div class="form__content">
+        <label for="input" class="icon"></label>
+        <input
+          class="input input--one"
+          type="text"
+          id="input"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </div>
+    </form>
+
+    <form
+      class="form form--place"
+      data-qa="small"
+      action="/"
+      method="post"
+      >
```

Comment:
```suggestion
    <form
      class="form form--place"
      data-qa="small"
      action="/"
      method="post"
    >
```
<img width="1052" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/70220667/a2582fe7-7f55-43eb-8f90-4ad8eb2ba7a3">